app/src/main/java/com/prangesoftwaresolutions/audioanchor/PlayActivity.java

```diff
@@ -765,20 +765,6 @@ void showPlaybackSpeedDialog() {
         builder.setView(dialogView);
 
         final TextView playbackSpeedTV = dialogView.findViewById(R.id.playback_speed_tv);
-        final ImageView resetIV = dialogView.findViewById(R.id.reset);
-        resetIV.setOnClickListener(v -> {
-            int speed = Integer.parseInt(getString(R.string.preference_playback_speed_default));
-            float speedFloat = (float) (speed / 10.0);
-            playbackSpeedTV.setText(getResources().getString(R.string.playback_speed_label, speedFloat));
-
-            // Store new playback speed in shared preferences
-            SharedPreferences.Editor editor = mSharedPreferences.edit();
-            editor.putInt(getString(R.string.preference_playback_speed_key), speed);
-            editor.apply();
-
-            // Set playback speed to speed selected by the user
-            mPlayer.setPlaybackSpeed(speedFloat);
-        });
```

Comment:
The diff here shows that something about all these lines was changed (my guess would be tabs or spaces or line endings, something like that). That should be fixed such that it is easy to see what the real change you made was (the extra line of code + comment).

Reply:
These lines have not been changed but moved at another place, after initialisation `playbackSpeedSB`, because moving slider use `playbackSpeedSB.setProgress()`. 
I have no idea how make these moved lines more readable.

Reply:
Oh right, my bad, that's ok then.